test/container.spec.ts

```diff
@@ -1,10 +1,10 @@
 import "expect-puppeteer";
-import crypto from 'crypto'
+import crypto from "crypto";
```

Comment:
@victor-meng Any idea why these weren't already formatted correctly?

Reply:
I believe @zfoster was looking into it for one of his PR's.

Reply:
@southpolesteve This file wasn't formatted correctly when Zach checked it in in this commit: https://github.com/Azure/cosmos-explorer/commit/3fe63e88cb581c802a48919afd8111dbefeb1469. I probably turned formatting off too when I was making changes to some CSS files when I also modified this file.